ludwig/features/base_feature.py

```diff
@@ -110,6 +121,9 @@ def __init__(self):
         self.probabilities_key = PROBABILITIES
         self.logits_key = LOGITS
 
+    def forward(self, *args, **kwargs):
```

Comment:
How is this not infinitely recursive?

Reply:
Hmm, good question! Shouldn't be needed now that we made it an nn.Module again. Removed!